### PR TITLE
Track1 sdk automation/azure mgmt kusto

### PR DESCRIPTION
**Pull request by changlong-liu**

### CODE REVIEW

File: sdk/kusto/azure-mgmt-kusto/azure/mgmt/kusto/version.py [hunk 1]
```diff
@@ -9,4 +9,4 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.7.0"
+VERSION = "0.7.1"
```
Comment (zikalino) [inline sdk/kusto/azure-mgmt-kusto/azure/mgmt/kusto/version.py:12]: versoin should be 0.8.0

Comment (changlong-liu) [reply sdk/kusto/azure-mgmt-kusto/azure/mgmt/kusto/version.py:12]: committed here: https://github.com/Azure/azure-sdk-for-python/pull/11774/commits/d52bf23e4bb0a39475a6f70061fb5bc7b451c921

Comment (changlong-liu) [reply sdk/kusto/azure-mgmt-kusto/azure/mgmt/kusto/version.py]: fixed here: https://github.com/Azure/azure-sdk-for-python/pull/11774/commits/5ca7343ae41581433cb56535cf2d00c54d8e7823

**Comment by changlong-liu** [comment]
move to here: https://github.com/Azure/azure-sdk-for-python/pull/11814

**Pull request closed without merging**